src/devices/philips_AJ3650.c

```diff
@@ -136,7 +136,7 @@ static int philips_callback(r_device *decoder, bitbuffer_t *bitbuffer)
     battery_status = packet[PHILIPS_PACKETLEN - 1] & 0x40;
 
     data = data_make(
-                     "model",         "",            DATA_STRING, _X("Philips-Temperature","Philips outdoor temperature sensor"),
+                     "model",         "",            DATA_STRING, _X("Philips-Temperature","Philips AJ3650 outdoor temperature sensor"),
```

Comment:
Maybe this needs to be changed to something like Philips-AJ3650-Temperature. This is also done for other devices, e.g. https://github.com/merbanan/rtl_433/blob/6b6b89321b0cca261819a6694dec21f30995b82c/src/devices/digitech_xc0324.c#L122
@zuckschwerdt should this be changed?

If this is left as-is, the decoder for the aj3650 and aj7010 willl both report the same `model`.